modules/packages/Sort.chpl

```diff
@@ -802,6 +868,307 @@ proc selectionSort(Data: [?Dom] ?eltType, comparator:?rec=defaultComparator)
     compilerError("selectionSort() requires 1-D array");
 }
 
+pragma "no doc"
+proc shellSort(Data: [?Dom] ?eltType, comparator:?rec=defaultComparator,
+               start=Dom.low, end=Dom.high)
+  where !Dom.stridable
```

Comment:
Should it be documented anywhere that radix sort does not support strided arrays?